.github/workflows/build-39-bluefin.yml

```diff
@@ -1,21 +1,11 @@
 name: Bluefin 39
 on:
-  merge_group:
   pull_request:
     branches:
       - main
       - testing
-    paths-ignore:
-      - '**.md'
-      - 'system_files/kinoite/**'
```

Comment:
I think you'll still want these paths-ignore so these do not build in PRs when you don't need them to?